external/select2/js/select2.js

```diff
@@ -0,0 +1,5746 @@
+/*!
```

Comment:
thoughts on using `npm` / `package.json` instead of manually including?
that'd make it easier to get security notifications via `npm audit` and github
would still need to bundle in svn, but that's fine

Reply:
Agree that we should use npm. Lets wait for wooSelect, we can this push this as a separate change.